### PR TITLE
feat: add DataVerification vote tx, query CLI

### PR DESCRIPTION
**Pull request by inchori**
- Implement `DataVerificationVote` Transaction and Query CLI.

### CODE REVIEW

File: x/datadeal/client/cli/txDataVerificationVote.go [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/gogo/protobuf/jsonpb"
+	"github.com/medibloc/panacea-core/v2/x/datadeal/types"
+	"github.com/spf13/cobra"
+)
+
+func CmdDataVerificationVote() *cobra.Command {
```
Comment (H4NLee) [inline x/datadeal/client/cli/txDataVerificationVote.go:14]: We usually name the tx cmd in the form of verbs. Also the name of transaction is `VoteDataVerification`.

```suggestion
func CmdVoteDataVerification() *cobra.Command {
```

**Review by H4NLee — APPROVED** [review]
Left some comments, but it's not that critical issues

File: x/datadeal/client/cli/query.go [hunk 2]
```diff
@@ -27,6 +27,7 @@ func GetQueryCmd(queryRoute string) *cobra.Command {
 	cmd.AddCommand(CmdGetDeals())
 	cmd.AddCommand(CmdGetDeal())
 	cmd.AddCommand(GetCmdDataSale())
```
[the remark this answers is not in the document]
Comment (inchori) [reply x/datadeal/client/cli/query.go:29]: 😂 I'll change the name of the command.

File: x/datadeal/client/cli/txDataVerificationVote.go (hunk 1 above)
Comment (inchori) [reply x/datadeal/client/cli/txDataVerificationVote.go:14]: Ah, I agree about that. BTW, I just refer to the `CmdRegisterOracleVote()` in `x/oracle` module.

Comment (H4NLee) [reply x/datadeal/client/cli/txDataVerificationVote.go:14]: I know. IMO, the name of that cmd should also be changed to `CmdVoteOracleRegistration`.

Comment (inchori) [reply x/datadeal/client/cli/txDataVerificationVote.go:14]: I'll change the `CmdRegisterOracleVote` to `CmdVoteOracleRegistration` in this branch.

**Review by audtlr24 — APPROVED** [review]
LGTM! 👍

**Review by gyuguen — COMMENTED** [review]
LGTM

**Pull request merged**